frame/evm/precompile/modexp/Cargo.toml

```diff
@@ -14,6 +14,7 @@ sp-io = { version = "3.0.0", default-features = false, git = "https://github.com
 fp-evm = { version = "1.0.0", default-features = false, path = "../../../../primitives/evm" }
 evm = { version = "0.25.0", default-features = false, features = ["with-codec"] }
 num = { version = "0.3", features = ["alloc"], default-features = false }
+log = "0.4"
```

Comment:
Needs to disable `std` by default.

Reply:
Is this crate actually used? If not, please remove it!

Reply:
Good catch. Removed in 5ed6577